__tests__/app.test.js

```diff
@@ -171,22 +176,44 @@ describe('GET /api/users', () => {
 			.expect(200)
 			.then((response) => {
 				expect(response.body.users).toBeInstanceOf(Array);
-				expect(response.body.users.length).toBe(4);
-        response.body.users.forEach((user) => {
-					expect(user).toEqual({
+				expect(response.body.users).toEqual([
+					{ username: 'butter_bridge' },
+					{ username: 'icellusedkars' },
+					{ username: 'rogersop' },
+					{ username: 'lurker' },
+				]);
+				response.body.users.forEach((user) => {
+					expect(user).toMatchObject({
 						username: expect.any(String),
-						name: expect.any(String),
-						avatar_url: expect.any(String),
 					});
 				});
 			});
 	});
-  test('404 - Not found', () => {
+	test('404 - Not found', () => {
 		return request(app)
 			.get('/api/unexisting_path')
 			.expect(404)
 			.then((response) => {
 				expect(response.body.msg).toBe('Not found');
 			});
 	});
+});
+describe('GET /api/articles/:article_id (comment count)', () => {
+	test('200 - Success : get the number of comment /api/articles/1 has', () => {
+		return request(app)
+			.get('/api/articles/1')
+			.expect(200)
+			.then((response) => {
+				expect(response.body.article).toEqual({
+					article_id: 1,
+					title: 'Living in the shadow of a great man',
+					topic: 'mitch',
+					author: 'butter_bridge',
+					created_at: '2020-07-09T20:11:00.000Z',
+					body: 'I find this existence challenging',
+					votes: 100,
+					comment_count: 11,
+				});
+			});
+	});
```

Comment:
Nice test! Can also check that selecting an article with no comments gives a 200.

Need some error testing here, 404 for article not found and 400 for invalid article id :)